.github/workflows/build-assets.yml

```diff
@@ -136,32 +138,33 @@ jobs:
-            ${{ secrets.DOCKER_USERNAME }}/build-${{ matrix.os }}:latest
-          cache-from: type=registry,ref=hpccbuilds/platform-build-${{ matrix.os }}:${{ needs.release.outputs.community_branch }}
-          cache-to: type=registry,ref=hpccbuilds/platform-build-${{ matrix.os }}:${{ needs.release.outputs.community_branch }},mode=max
+#      - name: Create Build Image
+#        uses: docker/build-push-action@v3
+#        with:
+#          builder: ${{ steps.buildx.outputs.name }}
+#          file: ${{ needs.release.outputs.folder_platform }}/dockerfiles/vcpkg/${{ matrix.os }}.dockerfile
+#          context: ${{ needs.release.outputs.folder_platform }}/dockerfiles/vcpkg
+#          push: true
+#          build-args: |
+#            VCPKG_REF=${{ steps.vars.outputs.vcpkg_sha_short }}
+#          tags: |
+#            ${{ secrets.DOCKER_USERNAME }}/build-${{ matrix.os }}:${{ needs.release.outputs.community_tag }} 
+#            ${{ secrets.DOCKER_USERNAME }}/build-${{ matrix.os }}:latest
+#          cache-from: type=registry,ref=hpccbuilds/platform-build-${{ matrix.os }}:${{ needs.release.outputs.community_branch }}
+#          cache-to: type=registry,ref=hpccbuilds/platform-build-${{ matrix.os }}:${{ needs.release.outputs.community_branch }},mode=max
 
       - name: CMake Packages
         if: ${{ !matrix.container && !matrix.ln }}
         run: |
           mkdir -p ${{ needs.release.outputs.folder_build }}
+          echo "${{ secrets.SIGNING_SECRET }}" > ${{ needs.release.outputs.folder_build }}/private.key
           plugins=("CASSANDRAEMBED" "COUCHBASEEMBED" "ECLBLAS" "H3" "JAVAEMBED" "KAFKA" "MEMCACHED" "MYSQLEMBED" "NLP" "REDIS" "SQLITE3EMBED" "SQS" "PLATFORM")
           for plugin in "${plugins[@]}"; do
             sudo rm -f ${{ needs.release.outputs.folder_build }}/CMakeCache.txt
             sudo rm -rf ${{ needs.release.outputs.folder_build }}/CMakeFiles
             docker_label=${{ secrets.DOCKER_USERNAME }}/build-${{ matrix.os }}:${{ needs.release.outputs.community_tag }}
-            docker run --rm --mount ${{ needs.release.outputs.mount_platform }} --mount ${{ needs.release.outputs.mount_build }} $docker_label "cmake -S /hpcc-dev/HPCC-Platform -B /hpcc-dev/build ${{ needs.release.outputs.cmake_options }} -D$plugin=ON -DCONTAINERIZED=OFF -DCPACK_STRIP_FILES=OFF" 
-            docker run --rm --mount ${{ needs.release.outputs.mount_platform }} --mount ${{ needs.release.outputs.mount_build }} $docker_label "cmake --build /hpcc-dev/build --parallel $(nproc) --target package" 
+            docker run --rm --mount ${{ needs.release.outputs.mount_platform }} --mount ${{ needs.release.outputs.mount_build }} $docker_label "${{ needs.release.outputs.load_key }} && cmake -S /hpcc-dev/HPCC-Platform -B /hpcc-dev/build ${{ needs.release.outputs.cmake_options }} -DSIGN_MODULES_PASSPHRASE=${{ secrets.SIGNING_PASSPHRASE }} -DSIGN_MODULES_KEYID=${{ secrets.SIGNING_KEYID }} -D$plugin=ON -DCONTAINERIZED=OFF -DCPACK_STRIP_FILES=OFF" 
```

Comment:
Any reason not to have `-DSIGN_MODULES_PASSPHRASE=${{ secrets.SIGNING_PASSPHRASE }} -DSIGN_MODULES_KEYID=${{ secrets.SIGNING_KEYID }}` in the global cmake_options, given SIGN_MODULE is there?